Tests/NIOTests/EventLoopTest.swift

```diff
@@ -144,7 +131,7 @@ public class EventLoopTest : XCTestCase {
         let expect = expectation(description: "Is cancelling RepatedTask")
         let group = DispatchGroup()
         group.enter()
-        let repeatedTask = eventLoopGroup.next().scheduleRepeatedTask(initialDelay: initialDelay, delay: delay) { () -> Void in
+        let repeatedTask = eventLoopGroup.next().scheduleRepeatedTask(initialDelay: initialDelay, delay: delay) { _ -> Void in
```

Comment:
Can you ignore this value but specify its type, please? Same with the two changes below.